spring-restdocs-core/src/test/java/org/springframework/restdocs/cli/HttpieRequestSnippetTests.java

```diff
@@ -149,7 +149,7 @@ public void postRequestWithQueryStringWithNoValue() throws IOException {
 	@Test
 	public void postRequestWithOneParameter() throws IOException {
 		this.snippets.expectHttpieRequest().withContents(codeBlock("bash")
-				.content("$ http --form POST 'http://localhost/foo' 'k1=v1'"));
+				.content(String.format("$ http --form POST 'http://localhost/foo' \\%n 'k1=v1'")));
```

Comment:
I'd prefer that the existing tests were largely unaffected by the introduction of the CommandFormatter. That could be done by leaving the content assertions as they were, but creating the snippet with a single-line formatter.